### PR TITLE
Break XLA flags into multiple lines to improve version history and allow easy revert

### PR DESCRIPTION
**Pull request by yhtang**
This way, each XLA flags has its own line in the Dockerfile and can be more easily added or removed.

### CODE REVIEW

File: .github/container/Dockerfile.jax [hunk 1]
```diff
@@ -55,13 +55,21 @@ ARG BUILD_DATE
...
 ENV CUDA_DEVICE_MAX_CONNECTIONS=1
 ENV NCCL_IB_SL=1
 ENV NCCL_NVLS_ENABLE=0
 ENV CUDA_MODULE_LOADING=EAGER
+## The following environment variables are WARs for known issues
+## Remove ASAP once the underlying issue has been removed
+# https://github.com/NVIDIA/JAX-Toolbox/pull/472, WAR for a crash observed in the default runtime leading to this error message: "nccl_all_reduce_thunk.cc:175] Check failed: reduction_kind.has_value()"
+ENV XLA_FLAGS="${XLA_FLAGS} --xla_gpu_enable_xla_runtime_executable=true"
+# https://github.com/NVIDIA/JAX-Toolbox/pull/487, WAR for the following LLVM crash observed in Pax: "Invalid LLVM IR before optimizations"
+ENV XLA_FLAGS="${XLA_FLAGS} --xla_gpu_enable_llvm_module_compilation_parallelism=true"
```
Comment (ashors1) [inline .github/container/Dockerfile.jax:72]: `--xla_gpu_enable_llvm_module_compilation_parallelism=true` has been removed in upstream JAX-toolbox. Can we rebase this PR?

Comment (yhtang) [reply .github/container/Dockerfile.jax:72]: Sure.

Comment (yhtang) [reply .github/container/Dockerfile.jax]: I just rebased. Could you please examine if the flags are up-to-date now?

Comment (ashors1) [reply .github/container/Dockerfile.jax]: Looks good, thanks!

**Review by ashors1 — APPROVED** [review]

**Pull request merged**